### PR TITLE
[Wait for #2233][Wait for #2265][Application] incremental inference pico gpt in android

### PR DESCRIPTION
**Pull request by lhs8928**
[Application] apply incremental inference to pico gpt
    
     - Apply incremental inference to pico gpt
    
    Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

File: Applications/Android/PicoGPTJNI/app/src/main/jni/encoder.hpp [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    else {
+      for (size_t i = 0; i < word.size() - 1; ++i) {
+        ranks[i] = m_bpe_ranks.find({word[i], word[i + 1]});
+      }
+      ranks.resize(word.size() - 1);
+    }
+  }
+
+  return word;
+}
```
Comment (SeoHyungjun) [inline Applications/Android/PicoGPTJNI/app/src/main/jni/encoder.hpp:393]: A new line is required.

File: Applications/PicoGPT/jni/encoder.hpp [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    else {
+      for (size_t i = 0; i < word.size() - 1; ++i) {
+        ranks[i] = m_bpe_ranks.find({word[i], word[i + 1]});
+      }
+      ranks.resize(word.size() - 1);
+    }
+  }
+
+  return word;
+}
```
Comment (SeoHyungjun) [inline Applications/PicoGPT/jni/encoder.hpp:393]: same issue

**Review by SeoHyungjun — APPROVED** [review]
need to change the order of headers. (ci clang)
LGTM

**Comment by jijoongmoon** [comment]
it includes in #2286

**Pull request closed without merging**